egs/fisher_swbd/s5/local/chain/run_tdnn_lstm_1b.sh

```diff
@@ -108,7 +108,7 @@ dir=${dir}$suffix
 build_tree_train_set=train_nodup
 train_set=train_nodup_sp
 build_tree_ali_dir=exp/tri5a_ali
-treedir=exp/chain/tri6_tree
+treedir=exp/chain/tri5_tree
```

Comment:
I think you should change this  line back because it could problems when running from a later --stage.